src/cli/self_update.rs

```diff
@@ -189,6 +195,7 @@ To configure your current shell run `source {cargo_home}/env`
     };
 }
 
+#[cfg(windows)]
```

Comment:
would you like this reviewed, or are you planning on moving these now windows only things into the windows self-install module too ?

Reply:
This is where it currently stands. Moving things around further is out-of-scope. I only performed what seemed like the most obvious changes.

Reply:
A review would be appreciated. In general, I am trying to deliberately block myself from pushing further changes in to a diff. I would rather split the work and submit a followup than have the entire thing turned down, and I would rather have less things to rebase.